docs/Submitting_jobs.rst

```diff
@@ -33,19 +33,22 @@ Selecting the job backend (``--job-backend``)
 
 The job backend to be used can be specified using the ``--job-backend`` EasyBuild configuration option.
 
-Since EasyBuild 2.2.0, two backends are supported:
+Since EasyBuild 3.8.0, three backends are supported:
 
-* ``GC3Pie`` *(default, recommended)*
+* ``GC3Pie`` *(default)* (supported since EasyBuild 2.2.0)
 
   * ``GC3Pie`` version 2.4.0 (or more recent) required (https://gc3pie.readthedocs.org)
-  * works with different resource managers and job schedulers, including TORQUE, SLURM, etc.
+  * works with different resource managers and job schedulers, including TORQUE/PBS, Slurm, etc.
   * **note**: requires that a GC3Pie configuration file is provided, see :ref:`submitting_jobs_cfg_job_backend_config`
 
 * ``PbsPython``
 
   * ``pbs_python`` version 4.1.0 (or more recent) required (see https://oss.trac.surfsara.nl/pbs_python)
-  * **note**: requires TORQUE resource manager (see http://www.adaptivecomputing.com/products/open-source/torque/)
+  * **note**: requires Torque resource manager (see http://www.adaptivecomputing.com/products/open-source/torque/)
```

Comment:
Why the change to "Torque" ? Their web page uses TORQUE, and it's still TORQUE above for GC3Pie